ovirtlib4/hosts.py

```diff
@@ -31,7 +31,7 @@ def _get_collection_entity(self):
         return HostEntity(connection=self.connection)
 
 
-class HostEntity(CollectionEntity):
+class HostEntity(CollectionEntity, ClusterEntity):
```

Comment:
Not all entities have a cluster herf, so I add ClusterEntity to the one that supports it, if you know about any other entity that have cluster href, pls let me know

Reply:
I do understand it can be correct and beneficial to have a cluster entity representation for entities that are associated with clusters, but I don't think inheritance is the solution here.
Using inheritance you're adding a lot of boilerplate code which is cluster management related to entities that should not have such options.
Is it ok for a host to be able to change network roles in a cluster? to be able to change MAC pool range?
Please reconsider aggregation here, since inheritance can have bad influences when used out of context.